doc/multiprecision.qbk

```diff
@@ -1091,57 +1108,85 @@ the default by defining either `BOOST_MP_USE_FLOAT128` or `BOOST_MP_USE_QUAD`.
 * When the underlying type is Intel's `_Quad` type, the code must be compiled with the compiler option `-Qoption,cpp,--extended_float_type`.
 * When compiling with `gcc`, you need to use the flag `--std=gnu++11/14/17`, as the suffix 'Q' is a GNU extension. Compilation fails with the flag `--std=c++11/14/17`
 unless you also use `-fext-numeric-literals`.
+* You will need to link to `libquadmath.dll` with the link command `-lquadmath` and ensure that the DLL is visible by the linker. 
```

Comment:
I worry this is too windows centric?

Reply:
> I worry this is too windows centric?

As it stands, it's not quite correct because you will not be linking to the DLL.

One option would be to use a more vague, yet very simplified generic form:

- You need to link with `quadmath` and also ensure that symbols from the quadmath library are available.

Another option would be to describe more precisely for Win* or *nix separately.

- On `*nix` systems, you need to link with `quadmath` by providing the linker flag `-lquadmath`, which will link with `libquadmath.a`.
- On `Win*` systems such as GCC ported to MinGW, you also need to provide the linker with the `-lquadmath` flag in order to link with `libquadmath.lib`. It might also be necessary to ensure that symbols from the quadmath DLL are available at runtime.

Reply:
Perfect!  Glad one of us has a brain this evening! :)